boards/default/distros/br/br.py

```diff
@@ -164,11 +189,43 @@ def configure(self, env):
         wlutil.run([mergeScript] + kFrags, cwd=(br_dir / 'buildroot'), env=env)
 
     # Build a base image in the requested format and return an absolute path to that image
-    def buildBaseImage(self):
+    def buildBaseImage(self, task, changed):
```

Comment:
Can you add a type annotation and a docstring on what task, changed are? I also couldn't find where this is being passed into this function

Reply:
Sure. They are passed by `doit` to the function. I can add some doc on this.